src/views/bouquets/BouquetEditView.vue

```diff
@@ -53,7 +160,12 @@ const onSubmit = async () => {
   }
 }
 
+watch(selectedThematique, (theme) => {
```

Comment:
Maybe rather a change event on the select field?

Reply:
I made the changes

Reply:
I changed it for an event